files/bootstrap.sh

```diff
@@ -20,6 +20,7 @@ function print_help {
     echo "--b64-cluster-ca The base64 encoded cluster CA content. Only valid when used with --apiserver-endpoint. Bypasses calling \"aws eks describe-cluster\""
     echo "--apiserver-endpoint The EKS cluster API Server endpoint. Only valid when used with --b64-cluster-ca. Bypasses calling \"aws eks describe-cluster\""
     echo "--kubelet-extra-args Extra arguments to add to the kubelet. Useful for adding labels or taints."
+    echo "--kubelet-extra-json-args Extra json configuration for kubelet. Userful for adding all kinds of extra configuration such as evictionHard or kubeReserved settings in JSON format."
```

Comment:
Can we also add `Will override any conflicting setting that the script defaults to.`